docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,667 @@
+Our observations of the particle's position are subject to noise.
+In this section, we model this as Gaussian additive noise on the positions in the track.
+
+Note that free diffusion assumes that the space available for diffusion :math:`L` is sufficiently large that the measurement time interval is much smaller than :math:`L / 2D` :cite:`qian1991single`:
+
+To simulate diffusive tracks with observation noise, we can use :func:`~lumicks.pylake.simulation.simulate_diffusive_tracks`::
+
+    tracks = lk.simulation.simulate_diffusive_tracks(diffusion_constant=1.0, steps=40, dt=0.01, observation_noise=0.1, num_tracks=25)
+    tracks.plot()
+
+.. image:: figures/simulated_tracks.png
+
+We will use this functionality to demonstrate the various methods for estimating diffusion constants implemented in Pylake.
+
+Method background
+-----------------
+
+Consider a freely diffusive track, how would one quantify its diffusivity?
+One convenient way would be to consider its positional increments over time.
+As shown in the simulation model, the increments in free diffusion are independent of the current position and depend directly on the diffusion constant.
+Considering that diffusion is a stochastic process, it is important to average over many such increments to obtain a result not subject to a lot of stochastic error.
+The exact details of how to obtain a diffusion constant depend on the method in question and will be discussed below.
+
+Mean Squared Displacements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simply averaging the increments will yield an estimate of zero (since the original increment distribution had a mean of zero).
+Instead, a measure of distance travelled is required.
+One easy way to do this is to square these displacements prior to averaging.
+When dealing with pure diffusive motion (a complete absence of drift) in an isotropic medium, the averaged squared displacements follow the following relation:
+
+.. math::
+
+    \rho[n] = \langle\left(x_{i} - x_{i + n}\right)^2\rangle = 2 D n \Delta t
+
+Here the brackets :math:`\langle\ldots\rangle` indicate the expected value, :math:`\rho[n]` corresponds to the average squared displacement for lag :math:`n` and :math:`D` is the diffusion constant we seek.
+This is the reason diffusive motion is often quantified using Mean Squared Displacements (MSD).
+There are multiple ways to estimate this quantity in practice.
+We can average over different tracks (ensemble averaging) or over time (time averaging).
+If all tracks experience the same diffusion, the ensemble averaged motion of a collection of independent molecules results in the same diffusion estimates as the time-averaged motion.
+Considering that we may be interested in differences in diffusion constants between different tracks, we use the following time-averaged estimator:
```

Comment:
```suggestion
If the diffusive process is not ergodic, and the goal of the analysis is to find potential differences in diffusion constants between different tracks, the following time-averaged estimator can be used:
```

Reply:
I think the inclusion of ergodicity can lead to confusion here.

Pure diffusion as described in this chapter is ergodic.

You can have sub-populations with different diffusion constants, but that doesn't change that the diffusion itself is ergodic. If you were able to isolate the two distributions and average a _lot_ of data, you'd see that EAMSD gives the same result as EATAMSD. This is something that is true for pure diffusion and actually something they test for in some types of anomalous diffusion (whether there is ergodicity breaking).

I do like the other changes to the sentence though. I would suggest going with the following instead if you're also ok with that:

`If the goal of the analysis is to find potential differences in diffusion constants between different tracks, the following time-averaged estimator can be used:`

Reply:
Good point, I like your suggestion.